src/test/kotlin/org/rust/lang/core/type/RsTypificationTestBase.kt

```diff
@@ -84,3 +89,7 @@ abstract class RsTypificationTestBase : RsTestBase() {
         }
     }
 }
+
+private val BUILTIN_MACRO_NAMES = listOf(
+    "env", "option_env", "concat", "line", "column", "file", "stringify", "module_path", "cfg"
```

Comment:
It's exactly this list: https://github.com/intellij-rust/intellij-rust/blob/master/src/main/kotlin/org/rust/lang/core/types/infer/TypeInferenceWalker.kt#L1077

We need it because such built-in macros actually resolve to dummy definitions that expand to empty block expr (`{ }`) that we don't take into account during type inference.